tools/repo/notify.py

```diff
@@ -34,43 +34,42 @@
 SLACK_EXPORT_URL = "https://api.slack.com/apps/A023NPQQ33K/oauth?"
 
 MAINTAINERS = {
+    'abeyad': 'U03CVM7GPM1',
+    'adisuissa': 'UT17EMMTP',
     'alyssawilk': 'U78RP48V9',
-    'mattklein123': 'U5CALEVSL',
-    'lizan': 'U79E51EQ6',
-    'snowp': 'U93KTPQP6',
     'ggreenway': 'U78MBV869',
     'htuch': 'U78E7055Z',
-    'zuercher': 'U78J72Q82',
-    'phlax': 'U017PLM0GNQ',
     'jmarantz': 'U80HPLBPG',
-    'ravenblackx': 'U02MJHFEX35',
-    'yanavlasov': 'UJHLR5KFS',
-    'RyanTheOptimist': 'U01SW3JC8GP',
-    'adisuissa': 'UT17EMMTP',
     'KBaichoo': 'U016ZPU8KBK',
-    'wbpcode': 'U017KF5C0Q6',
-    'kyessenov': 'U7KTRAA8M',
     'keith': 'UGS5P90CF',
-    'abeyad': 'U03CVM7GPM1',
+    'kyessenov': 'U7KTRAA8M',
+    'lizan': 'U79E51EQ6',
+    'mattklein123': 'U5CALEVSL',
+    'nezdolik': 'UDYUWRL13',
+    'phlax': 'U017PLM0GNQ',
+    'ravenblackx': 'U02MJHFEX35',
+    'RyanTheOptimist': 'U01SW3JC8GP',
+    'snowp': 'U93KTPQP6',
     'soulxu': 'U01GNQ3B8AY',
+    'wbpcode': 'U017KF5C0Q6',
+    'yanavlasov': 'UJHLR5KFS',
+    'zuercher': 'U78J72Q82',
 }
 
 # First pass reviewers who are not maintainers should get
 # notifications but not result in a PR not getting assigned a
 # maintainer owner.
 FIRST_PASS = {
-    'silverstar194': 'U03LNPC8JN9',
-    'nezdolik': 'UDYUWRL13',
-    'daixiang0': 'U020CJG6UU8',
     'botengyao': 'U037YUAK147',
+    'daixiang0': 'U020CJG6UU8',
+    'silverstar194': 'U03LNPC8JN9',
     'tyxia': 'U023U1ZN9SP',
 }
 
 # Only notify API reviewers who aren't maintainers.
 # Maintainers are already notified of pending PRs.
 API_REVIEWERS = {
     'markdroth': 'UMN8K55A6',
-    'adisuissa': 'UT17EMMTP',
```

Comment:
do we now have only one person as api reviewer?

Reply:
i was going on the comment - reasonably confident that last time i refactored the related code it respected existing way - so my reading is that there is no need to add in API_REVIEWERS

Reply:
I'm not an expert on the tooling, so I've got 2 questions:
1. will this impact the the PR notifier, and if so, how?
2. will this impact the auto-assigner of the api-reviewers?

Reply:
i believe the answer is no to both questions - but i can track through source and check

Reply:
wrt the 2nd question - totally different system https://github.com/envoyproxy/envoy/blob/b483d296738516839ec09ea3056f9c90be362080/repokitteh.star#L11-L20

Reply:
wrt 1st - reading source again im pretty confident it has no effect - its only used to notify personally and all maintainers are notified whenever they are assigned anyway (as per comment)

lmk if for some reason this is not correct